### PR TITLE
[Azure resource graph] ignore subscription from context

### PR DESCRIPTION
**Pull request by OlhaTkachenko**
## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs [hunk 1]
```diff
@@ -224,16 +224,6 @@ private IEnumerable<string> GetSubscriptions()
                 return this.Subscription;
             }
 
-            // Use selected subscription (for example, by command "Select-AzSubscription {subscriptionId}") 
-            if (this.TryGetDefaultContext(out var context))
```
Comment (cormacpayne) [inline src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs]: @OlhaTkachenko are we able to add a test for this scenario to ensure that we're returning the correct list of subscriptions when the user doesn't provide a value for the `-Subscription` parameter?

**Comment by cormacpayne** [comment]
@OlhaTkachenko this PR is currently targeted against the `release-2019-08-27` branch -- please switch this to the `master` branch and make sure to pull the latest from `master` into your branch.

File: src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs (hunk 1 above)
Comment (OlhaTkachenko) [reply src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs]: During test, it is uses subscription from DefaultAccount from SetupAzureEnvironmentFromEnvironmentVariables()

**Comment by chiragg4u** [comment]
Since our service is GA now, can we release this as 1.0 version?

**Pull request merged**